Block/Dom/Image.php

```diff
@@ -2,10 +2,20 @@
 
 namespace Elgentos\PrismicIO\Block\Dom;
 
-class Image extends Raw
+use Elgentos\PrismicIO\Block\AbstractBlock;
+
+class Image extends AbstractBlock
 {
     public function fetchDocumentView(): string
     {
-        return '<img src="' . $this->escapeHtml(parent::fetchDocumentView()) . '" alt="' . $this->getAltText() . '" />';
+        $context = $this->getContext();
+
+        $cssClasses = ($this->getData('css_class') ? 'class="'. $this->getData('css_class') .'"' : '');
+
+        return '<img src="' . $context->url . '"
```

Comment:
I'd use `sprintf()` here as it will make this easier to read:

```php
return sprintf(
    '<img src="%s" alt="%s" width="%d" height="%d" %s />',
    $context->url,
    $context->alt,
    $context->dimensions->width,
    $context->dimensions->height,
    $cssClasses
);
```

With `%d` you can ensure the value is always an integer.